src/plugins/telemetryTable/TelemetryTableConfiguration.js

```diff
@@ -46,6 +46,10 @@ define([
         getConfiguration() {
             let configuration = this.domainObject.configuration || {};
             configuration.hiddenColumns = configuration.hiddenColumns || {};
+            configuration.columnWidths = configuration.columnWidths || {};
```

Comment:
this should be matched with https://github.com/nasa/openmct/blob/91629149a6a0f4a65d18ae6e430b7519b186677a/src/plugins/telemetryTable/TelemetryTableType.js#L32

Reply:
👍